frontend/awx/administration/notifiers/hooks/useNotifiersFilters.tsx

```diff
@@ -0,0 +1,10 @@
+import { useDynamicToolbarFilters } from '../../../common/useDynamicFilters';
+
+export function useNotifiersFilters() {
+  const toolbarFilters = useDynamicToolbarFilters({
+    optionsPath: 'notification_templates',
+    preSortedKeys: ['name'],
+    preFilledValueKeys: ['name', 'id'],
```

Comment:
The old UI has additional filters: Created by and Modified by. We should add these as well.

Reply:
Good catch!!